### PR TITLE
[Docs] Add AI chatbot to the docs site

### PR DESCRIPTION
**Pull request by august-growthbook**
### Features and Changes
We recently started using Kapa.ai's AI chat capabilities to help with our customer support tickets. Kapa.ai also offers a chat widget that lives within a documentation site. This PR adds that chat widget to the GrowthBook docs site.

### Dependencies
None

### Testing
1. Pull my branch to your machine. 
2. Navigate to the `docs` directory (`$ cd docs`)
3. Run `$ yarn dev`
4. Navigate to http://localhost:3200/
5. Confirm the chat widget appears in the lower right corner (shown below)
6. Click on the chat widget and use the search bar to type in a question. The chat bot should return an answer based on our docs site.

### Screenshots

**EDITED on 12/20/2023** to replace screenshot. The old one had a GrowthBook icon; the new one below has a chatbot icon.

Screen Shot 2023-12-20 at 10 52 04 AM

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -24,6 +24,20 @@ const config = {
...
+  // Kapa.ai chat bot on Docs page
+  scripts: [
+    {
+      src: "https://widget.kapa.ai/kapa-widget.bundle.js",
+      "data-website-id": "c4406b9f-35c5-43ca-b0c1-e7c0e261831f",  // Safe to expose publicly
+      "data-project-name": "GrowthBook",
+      "data-project-color": "#7817d3",
+      "data-modal-example-questions":"How do I run an experiment?",
+      "data-project-logo":
+        "https://raw.githubusercontent.com/growthbook/growthbook/main/docs/static/images/white-growthbook-icon-purple-bg-256.png",
```
Comment (Auz) [inline docs/docusaurus.config.js:36]: We shouldn't link to github's url for this file - use the docs URL (starting with docs.growthbook.io...)

File: docs/static/images/white-growthbook-icon-purple-bg-256.png
Comment (Auz) [inline docs/static/images/white-growthbook-icon-purple-bg-256.png:1]: chatbot-icon-white
What do we think of something like this?

Comment (august-growthbook) [reply docs/static/images/white-growthbook-icon-purple-bg-256.png:1]: Sure 👍🏻 I think we should still keep the "Ask AI" text below the icon so it's very clear what the widget is for. It doesn't have to be "Ask AI" specifically, something else could work as well. Is that what you had in mind, to keep the text and just replace the icon?

File: docs/docusaurus.config.js (hunk 1 above)
Comment (august-growthbook) [reply docs/docusaurus.config.js:36]: Understood, thanks 👍🏻

File: docs/static/images/white-growthbook-icon-purple-bg-256.png
Comment (august-growthbook) [reply docs/static/images/white-growthbook-icon-purple-bg-256.png:1]: I replaced the former GrowthBook icon with the new chatbot icon.

**Comment by august-growthbook** [comment]
@Auz This is ready for your review. The icon has been changed to the new chatbot icon.

**Review by Auz — APPROVED** [review]

**Pull request merged**